### ISSUE TITLE
Add symbols option for keycode mapping and convert modified keycodes to use them

### ISSUE THREAD

**Issue by caksoylar** [caksoylar/keymap-drawer#9]
If we commit to use unicode symbols in the keymap, e.g. for arrow keys or for modifiers (like `⌘` for GUI, `⇧` for shift), we can either default to those in the keycode conversion in `parse_config.*_bindings_map` or more likley add them as an alternative that can be toggled on or off (`parse_config.use_symbols: bool = True`).

We can also convert modified functions to automatically `<symbol>keycode` representations like https://github.com/infused-kim/zmk-config/blob/9f2ed72247120b9439df989bf1361a04f4646278/keymap_img/chocofi_keymap_config.yaml#L283. I think that might make @infused-kim's life easier as well!

**Comment by caksoylar** [comment]
This kind of configuration will probably be used for determining symbols for modifiers, as suggested in https://github.com/caksoylar/keymap-drawer/issues/16#issuecomment-1474691533:
```yaml
parse_config:
  mod_map:
    GUI: ⌘
    CTRL: ⌃
    ALT: ⎇
    SHIFT: ⇧
```
Then for instance `kp LS(LC(TAB))` can be automatically converted to `⇧⌃ TAB`.

**Comment by caksoylar** [comment]
A solution for this lives in the `dev` branch now at https://github.com/caksoylar/keymap-drawer/tree/dev, with documentation available in https://github.com/caksoylar/keymap-drawer/blob/dev/CONFIGURATION.md#modifier_fn_map. I am not 100% happy with the new default behavior so I didn't merge it, any feedback is welcome. You can also test this using https://keymap-drawer-dev.streamlit.app.

**Comment by caksoylar** [comment]
Resolved by b5bf711039a787c1f5e22ab2de2eaf071fbac7c1, released in v0.16.0.
